sklearn/feature_selection/_sequential.py

```diff
@@ -175,18 +181,34 @@ def fit(self, X, y):
             self.n_features_to_select_ if self.direction == 'forward'
             else n_features - self.n_features_to_select_
         )
-        for _ in range(n_iterations):
-            new_feature_idx = self._get_best_new_feature(cloned_estimator, X,
-                                                         y, current_mask)
-            current_mask[new_feature_idx] = True
 
-        if self.direction == 'backward':
-            current_mask = ~current_mask
+        if self.aborted_rate is None:
+            for _ in range(n_iterations):
+                new_feature_idx, new_score = self._get_best_new_feature_score(cloned_estimator, X,
+                                                             y, current_mask)
```

Comment:
code style (PEP8):

```suggestion
                new_feature_idx, new_score = self._get_best_new_feature_score(
                    cloned_estimator, X, y, current_mask
                )
```

Reply:
fix it on https://github.com/scikit-learn/scikit-learn/pull/20145/commits/2922dbba5e33e7b0e6990ba71cc33372307ca9a0

I tried using `autopep8`. It's so useful!
however, The position of the parentheses is different from the code @ogrisel  suggested ...
Should I fix it?